lib/components/fields/DescriptionField.js

```diff
@@ -34,7 +34,7 @@ function DescriptionField(props) {
       _context.ContextConsumer,
       null,
       function (markdownRenderer) {
-        return markdownRenderer(description);
+        return markdownRenderer(description.replace(/<br>/gi, "\n"));
```

Comment:
@Inam-urRehman You shouldn't need to do this here. The markdown renderer should be the one doing this.

By doing this, you're leaking Markdown renderer-specific behavior into the RJSF codebase.

Reply:
@thehappybug This is an existing issue. Our markdown renderer is not replacing `<br>`. If you want us to enhance markdown renderer then we should handle this in separate issue.

Reply:
This is an optional fix. **Our** Markdown renderer should be responsible for replacing `<br>` on its own.

PS: I'm not asking us to enhance the markdown renderer library we are using; just that our wrapper over it should be responsible for doing this.